src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -534,6 +538,31 @@ export default {
 		},
 	},
 
+	watch: {
+		isTemporaryUpload: {
+			immediate: true,
+			handler(value) {
+				this.uploadManager = value ? getUploader() : null
+			}
+		},
```

Comment:
It is temporary only in the beginning, why don't you call getUploader() when on mounted ? 
and you can remove it once upload is finished

Reply:
Maybe a better idea, also helps to not create the uploadManager for dialog window